modules/postgresql/outputs.tf

```diff
@@ -152,3 +152,11 @@ output "env_vars" {
     "CLOUD_SQL_DATABASE_NAME" : local.database_name
   }
 }
+
+output "apphub_service_uri" {
+  value = {
+    service_uri = "//cloudsql.googleapis.com/projects${element(split("/projects", google_sql_database_instance.default.self_link), 1)}"
+    service_id  = substr(format("%s-%s", var.name, md5(var.project_id)), 0, 63)
```

Comment:
Could you add location field to the output. Similar to https://github.com/GoogleCloudPlatform/terraform-google-cloud-run/blob/main/modules/v2/outputs.tf#L86
It goes for everywhere you are adding apphub_service_uri. Also update the type in the metadata.yaml to reflect new field.